src/ui/flutter_app/lib/screens/game_page/board.dart

```diff
@@ -51,9 +51,7 @@ class Board extends StatelessWidget {
           child: Text(
             squareDesc[index],
             style: const TextStyle(
-              color:
-                  EnvironmentConfig.devMode ? Colors.red : Colors.transparent,
-              // TODO: [Leptopoda] instead of making it transparent when not needed we should not show it in the first place
+              color: Colors.red,
```

Comment:
If Git commit message is `Only show grid for screen reader on dev mode`, I think it's clear.

Reply:
but that's already the case. Changing the commit message to your proposed one would imply that we currently also show it when the screen reader is disabled.

This really just is the performance improvement I explained earlier